roqoqo-iqm/src/interface.rs

```diff
@@ -155,6 +259,8 @@ pub fn call_circuit<'a>(
                                     None => return Err(RoqoqoBackendError::GenericError {
                                         msg: format!("Output register {} has not been initialized correctly.", &readout) })
                                 };
+                            // TODO we should check that readout_length >= number_qubits_in_circuit here
+                            // Currently there is no simple way to obtain the number of qubits in the roqoqo circuit
```

Comment:
In Rust this should be available, no?

Reply:
It looks like I can do `circuit.involved_qubits()` and then match the variants of `InvolvedQubits`, is this what you have in mind?

Reply:
Yes that's right